instrumentation/http-url-connection/src/main/java/io/opentelemetry/auto/instrumentation/httpurlconnection/UrlInstrumentation.java

```diff
@@ -85,19 +83,17 @@ public static void errorSpan(
         String protocol = url.getProtocol();
         protocol = protocol != null ? protocol : "url";
 
-        Span span = TRACER.spanBuilder(protocol + ".request").setSpanKind(CLIENT).startSpan();
-
+        Span span = TRACER.startSpan(protocol + ".request");
         try (Scope scope = currentContextWith(span)) {
           span.setAttribute(SemanticAttributes.HTTP_URL.key(), url.toString());
           span.setAttribute(
```

Comment:
This already happens inside Tracer, right?

Reply:
port is handled inside onPeerConnection.  It's not covered by default though. setPeer is only taking care of net.peer.name, net.peer.ip and peer.service.

Reply:
Add a todo in #912.  Need to update tests for this change.